### PR TITLE
add download button to estimate set summary

### PR DESCRIPTION
**Pull request by hillalex**
Adds a download button to the current estimate set summary. Only appears if the set exists and is non-empty.

### CODE REVIEW

File: app/src/integrationTests/ContribIntegrationTests.tsx [hunk 1]
```diff
@@ -374,9 +377,6 @@ class ContributionPortalIntegrationTests extends IntegrationTestSuite {
             const modelVersionId = await addModel(this.db);
             const setId = await addBurdenEstimateSet(this.db, responsibilityIds.responsibility, modelVersionId);
 
-            const value = 32156;
```
Comment (hillalex) [inline app/src/integrationTests/ContribIntegrationTests.tsx]: just noticed this wasn't used, and below

**Comment by hillalex** [comment]
> Looks good, just needs conflicts fixing.
> 
> Little cosmetic thing - button looks a little crowded by the right-aligned text on the main responsibilities page (looks fine on the upload page.
> Screenshot from 2019-08-09 13-24-17

Oh yeah, good point. I'm minded to remove the right-alignment on that text as a fix - would you agree?

**Comment by EmmaLRussell** [comment]
yes, I don't the r-align really does anything now that the button is there.

**Comment by hillalex** [comment]
> Looks good but seems to have a failing test now

Oops. This is an unrelated failure due to the change to the API I made. It's failing on master too now:
http://teamcity.montagu.dide.ic.ac.uk:8111/viewLog.html?buildId=56808&tab=buildResultsDiv&buildTypeId=montagu_deploy

Updated the test on this branch: https://github.com/vimc/montagu-webapps/pull/400/commits/8dfb0f704de3e942615a86484347e5fcec14322d

**Pull request merged**